index.html

```diff
@@ -20,15 +20,17 @@ <h1>Quiz-App</h1>
         >
           <path d="M18 24l-6-5.269-6 5.269v-24h12v24z" />
         </svg>
-        <span>QUESTION</span>
+        <span class="card__title">QUESTION:</span>
```

Comment:
Maybe an h2 has a little bit more semantic meaning, especially for the following question paragraph?